### PR TITLE
Tooltips for various tags in Term Details panel

### PR DESCRIPTION
**Pull request by drashti171299**
I have changed the code according to the suggestion of @bryan-brancotte . I have added tootips. let me know is there any change or not.

### CODE REVIEW

**Comment by drashti171299** [comment]
@bryan-brancotte  is there any mistake in code?

File: js/tree-edam-stand-alone.js [hunk 1]
```diff
@@ -444,7 +468,7 @@ function interactive_edam_browser(){
                 value=interactive_edam_uri(value[0], translate_uri_to_text);
             }
         }
-        $("<tr><th>"+name+"</th><td class=\""+id+"\">"+interactive_edam_uri(value, translate_uri_to_text)+"</td></tr>").appendTo(table);
+        $("<tr><th data-toogle=\"tooltip\" title=\""+tootip+"\">"+name+"</th><td class=\""+id+"\">"+interactive_edam_uri(value, translate_uri_to_text)+"</td></tr>").appendTo(table);
```
Comment (bryan-brancotte) [inline js/tree-edam-stand-alone.js:471]: there is a typo, it is not `data-toogle` but `data-toggle`

Comment (bryan-brancotte) [inline js/tree-edam-stand-alone.js:471]: You are using boostrap tooltip, but did not enabled them. I let you find how to do it. You can also decide to not use them, but in this case please justify it (in 1-2 sentences, I just want your opinion)

File: js/tree-edam-stand-alone.js [hunk 2]
```diff
@@ -248,27 +248,48 @@ function interactive_edam_browser(){
...
+            else{
                 append_row(table,entry,d.data[entry]);
+            }
+            }
         });
         var community = details.find("tbody.community");
         var caller_b=biotool_api().get_for(current_branch, d.data.text, uri, d);
         if (caller_b.is_enabled()){
-            var id_b = append_row(community,"<a target=\"_blank\" href=\"https://bio.tools\" title=\"Bioinformatics Tools and Services Discovery Portal\">bio.tools</a>","<i>loading</i>");
+            var id_b = append_row(community,"<a target=\"_blank\" href=\"https://bio.tools\">bio.tools</a>","<i>loading</i>","Bioinformatics Tools and Services Discovery Portal");
```
Comment (bryan-brancotte) [inline js/tree-edam-stand-alone.js:292]: good idea to also rework them so every title are created in only one place, even for existing code

**Review by bryan-brancotte — CHANGES_REQUESTED** [review]
Hi @drashti171299

I like the approach, there is a typo to fix and a choice to do, I let you continue this PR and commit needed changes.

File: js/tree-edam-stand-alone.js (hunk 1 above)
Comment (drashti171299) [reply js/tree-edam-stand-alone.js:471]: yes sure. I will try to look for this and enable bootstrap. Also, I will change this typo. It was my mistake.

File: js/tree-edam-stand-alone.js (hunk 2 above)
Comment (drashti171299) [reply js/tree-edam-stand-alone.js:292]: I didn't understand this. what change is required in this?

**Comment by drashti171299** [comment]
@bryan-brancotte I have updated the code. Have a look at that !

**Comment by drashti171299** [comment]
@bryan-brancotte  can you verify these changes and close this PR? Outreachy applications are due in few days.

File: js/tree-edam-stand-alone.js (hunk 2 above)
Comment (hmenager) [reply js/tree-edam-stand-alone.js:298]: I don't think any change is needed here, it was just an appreciation of your coding strategy :smile:

**Review by hmenager — APPROVED** [review]
Looks good to me, thank you @drashti171299!

File: js/tree-edam-stand-alone.js (hunk 1 above)
Comment (hmenager) [reply js/tree-edam-stand-alone.js]: fixed AFAIK

Comment (hmenager) [reply js/tree-edam-stand-alone.js]: fixed I guess

Comment (drashti171299) [reply js/tree-edam-stand-alone.js]: yes it is fixed now @hmenager.

File: js/tree-edam-stand-alone.js (hunk 2 above)
Comment (drashti171299) [reply js/tree-edam-stand-alone.js:298]: Thank you @hmenager. But if any change is needed, I am happy to do that

File: index.html [hunk 3]
```diff
@@ -28,6 +28,7 @@
     <script src="https://cdnjs.cloudflare.com/ajax/libs/d3/4.13.0/d3.min.js"
             integrity="sha256-hYXbQJK4qdJiAeDVjjQ9G0D6A0xLnDQ4eJI9dkm7Fpk=" crossorigin="anonymous"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/levenshtein/1.0.5/levenshtein.min.js"></script>
+
```
Comment (bryan-brancotte) [inline index.html:31]: not needed, please remove this line

File: css/index.css [hunk 4]
```diff
@@ -1,3 +1,6 @@
+th[data-toggle="tooltip"]{
+    cursor: pointer;
```
Comment (bryan-brancotte) [inline css/index.css:2]: Indeed the text pointer is not adapted, `pointer` is good, but i would rather use `default`

**Review by bryan-brancotte — CHANGES_REQUESTED** [review]
Hi @drashti171299 

I added some minor remarque on code submitted. I reviewed your code and the tooltip are not actually enabled, I let you find why. 

After enabled it I get that:
tooltip-99

So the roadmap is:
 * enable the tooltip
 * understand the UI glitch
 * fix the UI glitch

File: css/index.css (hunk 4 above)
Comment (drashti171299) [reply css/index.css:2]: Okay I will remove this

File: index.html (hunk 3 above)
Comment (drashti171299) [reply index.html:31]: the last one should be removed only right?

Comment (bryan-brancotte) [reply index.html:31]: indeed only the newly added line, the idea is to not add blank line/change format when not necessary

**Comment by drashti171299** [comment]
@bryan-brancotte I am still not able to figure out why the bootstrap tooltips are not being enabled or what mistake I am doing in the code. but rest of your suggestions, I have done that thing. Review the code once again and tell me any other change you want in the code.

**Review by bryan-brancotte — COMMENTED** [review]
The issue was that you call `$('[data-toggle="tooltip"]').tooltip("enable");` only once when the app start, but at this moment the element that will later appear on the right does not exist yet, and thus not tooltip can be enabled. For example see [here](https://github.com/edamontology/edam-browser/pull/99/files#diff-c11ae8cf18cba2e900b4eee3972122950380827115f0cfb873ad70ba03280aa6R343), you can see that once the row is added we trigger the popover for this row.

File: index.html (hunk 3 above)
Comment (bryan-brancotte) [reply index.html]: Hi, I only said to remove the newly added blank line, not the levenshtein !

Comment (drashti171299) [reply index.html]: Ohk sorry for misunderstanding @bryan-brancotte . I am correcting it.

File: js/tree-edam-stand-alone.js [hunk 5]
```diff
@@ -474,7 +474,7 @@ function interactive_edam_browser(){
                 value=interactive_edam_uri(value[0], translate_uri_to_text);
             }
         }
-        $("<tr><th data-toggle=\"tooltip\" title=\""+tootip+"\">"+name+"</th><td class=\""+id+"\">"+interactive_edam_uri(value, translate_uri_to_text)+"</td></tr>").appendTo(table);
+        $("<tr><th><a href='#' data-toggle=\"tooltip\" data-placement='top' title=\""+tootip+"\">"+name+"</a></th><td class=\""+id+"\">"+interactive_edam_uri(value, translate_uri_to_text)+"</td></tr>").appendTo(table);
```
Comment (bryan-brancotte) [inline js/tree-edam-stand-alone.js:477]: you fixed the flickering issue with moving the title to a sub elements :clap:

Comment (drashti171299) [reply js/tree-edam-stand-alone.js:477]: Okay @bryan-brancotte.

Comment (drashti171299) [reply js/tree-edam-stand-alone.js:477]: so the UI glitch is solved right?

**Review by bryan-brancotte — CHANGES_REQUESTED** [review]
Hi, I pushed where the enabling should be done in 6dc25f79dd06f9c5d29f34f3792f1b0a43b4f7cc, once the span used, we are good for merge

File: js/tree-edam-stand-alone.js (hunk 5 above)
Comment (bryan-brancotte) [reply js/tree-edam-stand-alone.js:472]: yes UI glitch solved

File: index.html (hunk 3 above)
Comment (bryan-brancotte) [reply index.html]: corrected

Comment (bryan-brancotte) [reply index.html]: corrected with https://github.com/edamontology/edam-browser/pull/99/commits/66a7b33a19a2800775ce400e6f3cb0424584573d

File: css/index.css (hunk 4 above)
Comment (bryan-brancotte) [reply css/index.css]: I was suggesting to set cursor to default, see https://github.com/edamontology/edam-browser/pull/99/commits/d18935203862221e3ee6cb0f061cbe5733f79440 for updated css

File: js/tree-edam-stand-alone.js (hunk 5 above)
Comment (drashti171299) [reply js/tree-edam-stand-alone.js:471]: okay you are right. as a user, I can also confused like why here is link

Comment (drashti171299) [reply js/tree-edam-stand-alone.js]: I have changed the code. Have a look at that @bryan-brancotte

**Comment by drashti171299** [comment]
@bryan-brancotte  if you want any change, let me know. and thanks for guiding me in enabling tooltip.

**Pull request merged**